Makefile

```diff
@@ -29,11 +29,13 @@ GO_TEST_LDFLAGS =
 REPODIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))/
 LINT_FILE_PATH = $(REPODIR).golangci.yml
 
+VERSION_FILE=VERSION
+
 SHELL = /bin/bash
 
 HUB ?= docker.io/apache
 PROJECT ?= skywalking-go
-VERSION ?= $(shell git rev-parse --short HEAD)
+GIT_VERSION ?= $(shell git rev-parse --short HEAD)
```

Comment:
You still have not fixed this issue, In the released source tar, there will not be a GitHub, so it will fail.

Reply:
Can we build the corresponding VERSION file in `create_source_release.sh` and package it into the src package?

Even if the user downloads and compiles through the src package, the correct VERSION file content can be read

Reply:
Yes. When releasing, the version information needs to be written into the release source tar so that users downloading the release tar hosted by Apache will already have the version information included.